### ISSUE TITLE
MQTT control battery current

### ISSUE THREAD

**Issue by a900a900a900** [dalathegreat/Battery-Emulator#522]
Would be a useful to provide an extra safety step for control of the battery through home assistant. Using a range of sensors to control then control the battery current

**Comment by Sh0tgun98** [comment]
Yes please! This would greatly help with power flow optimisation for inverters that don't have the ability to change charge current.

**Comment by dalathegreat** [comment]
This was implemented to MQTT some time ago. See Wiki for info on how to set charge/discharge limits via MQTT: https://github.com/dalathegreat/Battery-Emulator/wiki/MQTT#set_limits
